torch/csrc/distributed/rpc/profiler/server_process_global_profiler.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <torch/csrc/autograd/profiler.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+using namespace torch::autograd::profiler;
+
+// Process global profiler state.
```

Comment:
Shall we expand the comment to explain why we need this class, and where it is used?

Reply:
Adding.